sky/packages/sky/lib/src/widgets/snack_bar.dart

```diff
@@ -38,7 +38,7 @@ class SnackBarAction extends Component {
   }
 }
 
-class SnackBar extends Component {
+class SnackBar extends StatefulComponent {
```

Comment:
AnimatedComponent is older than AnimationPerformance, and may have become obsolete.

Maybe that's the way to go. But here's what I was envisioning with the Transitions work: all animations would be encapsulated in these Transitions type classes. Parent classes then use them instead of becoming stateful. That way, the widgets that need to be rebuilt every frame are pushed as far down in the stack as possible.